test/e2e/serial/tests/resource_accounting.go

```diff
@@ -296,56 +297,80 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload resourc
 				corev1.ResourceMemory: maxResourceType(*targetNodeNRTInitial, corev1.ResourceMemory),
 			}
 
-			// add a min ammount of resources just in case all the zones
-			// are equal so the pod wont fit on any of them
-			excessRes := corev1.ResourceList{
-				corev1.ResourceCPU:    resource.MustParse("100m"),
-				corev1.ResourceMemory: resource.MustParse("100M"),
-			}
+			//calculate base load on the target node
+			baseload, err := nodes.GetLoad(fxt.K8sClient, targetNodeName)
+			ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", targetNodeName)
+			By(fmt.Sprintf("computed base load: %s", baseload))
+
 			reqResources := maxResources.DeepCopy()
 
+			//substract the baselaod from the max resources on the target node
 			reqCpu := reqResources[corev1.ResourceCPU]
-			reqCpu.Add(excessRes[corev1.ResourceCPU])
+			reqCpu.Sub(baseload.CPU)
 			reqResources[corev1.ResourceCPU] = reqCpu
-
 			reqMem := reqResources[corev1.ResourceMemory]
-			reqMem.Add(excessRes[corev1.ResourceMemory])
+			reqMem.Sub(baseload.Memory)
 			reqResources[corev1.ResourceMemory] = reqMem
 
 			By("Padding all other candidate nodes")
 			var paddingPods []*corev1.Pod
 			for _, nodeName := range nrtCandidateNames.List() {
 				node := &corev1.Node{}
 				nodeKey := client.ObjectKey{Name: nodeName}
-				err := fxt.Client.Get(context.TODO(), nodeKey, node)
+				err = fxt.Client.Get(context.TODO(), nodeKey, node)
 				Expect(err).NotTo(HaveOccurred())
 
-				// Padding all the other nodes until only maxResources is free
-				// remember reqResources = maxReources + excessResources
-				// so we ensure out pod will not be scheduled on any other node.
-				padPod, err := makeNodePaddingPod(fxt.Namespace.Name, *node, maxResources)
-				if errors.Is(err, e2enrt.ErrNotEnoughResources) {
-					klog.Infof("Node %q has not enough resources without padding", nodeName)
-					continue
+				/*
+					1. make pod per numa considering the baseload (baselaod/len(numa) per pod): saturate the numa leaving only the baseload
+					2. create the pods on the numas
+					3. the test pod should request an amount of resources that are available on compute node level but not on a single numa,
+					   thus need to account for the baseload when running the test pod on the target node
+				*/
+
+				//calculate base load on nodeName
+				baseload, err := nodes.GetLoad(fxt.K8sClient, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing node load info for %q", nodeName)
+				By(fmt.Sprintf("computed base load: %s", baseload))
+
+				//get nrt info of the node
+				By(fmt.Sprintf("preparing target node %q to fit the test case", nodeName))
+				nrtInfo, err := e2enrt.FindFromList(nrtCandidates, nodeName)
+				ExpectWithOffset(1, err).ToNot(HaveOccurred(), "missing NRT info for %q", nodeName)
+
+				//split the base laod on all numa zones
```

Comment:
the code seems to be contradicting the comment. What's the intention here?

Reply:
not relevant anymore, was considering splitting the baseload between all numa-zones, not sure if that is valid especially when numas > 2 and baseload is relatively little. So I'll adjust that comment. Thanks for your note.